frontend/src/components/ViewTrips/save-trip-form-elements.js

```diff
@@ -112,9 +112,10 @@ function createMultiFormControl(refArr, placeholder, defaultEmailArr) {
  *                           input prop.
  * @param {string} formLabel Label/title for the form input.
  * @param {string} inputType Input type of the form.
- * @param {!React.RefObject} ref Ref attached to the values inputted in the form.
+ * @param {!React.RefObject|!Array<React.RefObject>} ref Ref attached to the
+ *     values inputted in the form.
```

Comment:
Is this specifically for the collaborators field? Is it worth mentioning in the docs?

Reply:
Yes it is. I will add a note for that for the `ref` and `defaultVal` params.